### PR TITLE
Update chip-build Dockerfile

### PR DESCRIPTION
**Pull request by kpschoedel**
#### Problem

`gh_report.py` doesn't run in the `chip-build` container.

#### Change overview

Add python packages `cxxfilt` `ghapi` `pandas`.

#### Testing

Local run of `gh_report.py` in the updated container.

### CODE REVIEW

**Review by saurabhst — APPROVED** [review]

**Review by woody-apple — APPROVED** [review]

**Review by bzbarsky-apple — CHANGES_REQUESTED** [review]
Need to bump the version number.

File: integrations/docker/images/chip-build/Dockerfile [hunk 1]
```diff
@@ -86,7 +86,7 @@ RUN set -x \
     && : # last line
 
 RUN set -x \
-    && pip3 install attrs coloredlogs PyGithub pygit future portpicker mobly click \
+    && pip3 install attrs coloredlogs PyGithub pygit future portpicker mobly click cxxfilt ghapi pandas \
```
Comment (bzbarsky-apple) [inline integrations/docker/images/chip-build/Dockerfile:89]: Needs version bump.

**Review by bzbarsky-apple — APPROVED** [review]
Approving (so this is not blocked on me) if the version is bumped or @andy31415 or @woody-apple say it doesn't need to be.

File: integrations/docker/images/chip-build/Dockerfile (hunk 1 above)
Comment (kpschoedel) [reply integrations/docker/images/chip-build/Dockerfile:89]: Done

**Review by yufengwangca — APPROVED** [review]

**Review by andy31415 — APPROVED** [review]

**Pull request merged**